arcgis-ios-sdk-samples/Display information/Query features with Arcade expression/QueryFeaturesArcadeExpressionViewController.swift

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class QueryFeaturesArcadeExpressionViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set the touch delegate.
+            mapView.touchDelegate = self
+        }
+    }
+    
+    /// The arcade expression evaluation operation.
+    var evaluateOperation: AGSCancelable?
+    
+    /// Make and load a map.
+    func makeMap() -> AGSMap {
+        // Create a portal item with the portal and item ID.
+        let portalItem = AGSPortalItem(portal: .arcGISOnline(withLoginRequired: false), itemID: "14562fced3474190b52d315bc19127f6")
+        // Make a map with the portal item.
+        let map = AGSMap(item: portalItem)
+        // Load the map.
+        map.load { _ in
+            // Set the visibility of all but the RPD Beats layer to false.
+            map.operationalLayers.forEach { layer in
+                if let currentLayer = layer as? AGSLayer {
+                    currentLayer.isVisible = currentLayer.name.contains("RPD Beats")
+                }
+            }
+        }
+        return map
+    }
+    
+    /// Evaluate the arcade expression for the selected feature at the map point.
+    func evaluateArcadeInCallout(for feature: AGSArcGISFeature, at mapPoint: AGSPoint) {
+        // Show progress hud.
+        UIApplication.shared.showProgressHUD(message: "Evaluating")
+        // Instantiate a string containing the arcade expression.
+        let expressionValue =
+        """
+        var crimes = FeatureSetByName($map, 'Crime in the last 60 days');
+        return Count(Intersects($feature, crimes));
+        """
+        // Create an arcade expression using the string.
+        let expression = AGSArcadeExpression(expression: expressionValue)
+        // Create an arcade evaluator with the arcade expression and an arcade profile.
+        let evaluator = AGSArcadeEvaluator(expression: expression, profile: .formCalculation)
+        guard let map = mapView.map else { return }
```

Comment:
The `else` part of this also needs to hide the progress HUD.  Alternatively, you could move the `showProgressHUD` to just before the async `evaluator.evaluate` line.